expyfun/stimuli/_tracker.py

```diff
@@ -239,6 +248,22 @@ def respond(self, correct):
                 if self._direction <= 0:
                     self._direction = 1
 
+        if self._repeat_limit == 'reversals':
+            if self._x[-1] == self._x_min:
```

Comment:
This conditional seems weird. Let's say we have someone who just did a trial at the floor (dashed line indicating the next level to which they cannot ever go, i.e. `x_min - 1`):
```
\
 \
-----
```
If they get it right, we'd want it to do this:
```
\  
 \/
-----
```
But it seems like in this case no special treatment is required, because it should already be counted as a reversal by the existing logic. And this seems perfectly fine from an experimental validity standpoint.

In case they get it wrong, however, the track should stay at the same level (not get harder):
```
\
 \_
-----
```
But I don't think that's what this does?